### ISSUE TITLE
Is Gulden compatible with Coinomi?

### ISSUE THREAD

**Issue by Redsandro** [Gulden/gulden-official#3]
If someone is technical enough to [add Gulden to the Coinomi multi-coin wallet](https://coinomi.com/AddingSupportForANewCurrency/), I'm positive it will be a good influence on Guldens' popularity.

_edit:_

---

__We need:__

* Statum server support
* [BIP44 listing](https://github.com/satoshilabs/docs/blob/master/slips/slip-0044.rst) (Hey, E-gulden is on there)

And the value of:

* `pubkey_address`
* `script_address`
* `genesis_hash`
  * [6c5d71a461b5bff6742bb62e5be53978b8dec5103ce52d1aaab8c6a251582f92](https://explorer.gulden.com/#/block/6c5d71a461b5bff6742bb62e5be53978b8dec5103ce52d1aaab8c6a251582f92)

**Comment by mjmacleod** [comment]
Thanks for your request, we certainly have no problems in the technical department, however it is my understanding that Coinomi is integrated with Shapeshift and therefore are only likely to add coins that are also on Shapeshift. As you may know Gulden is not currently on Shapeshift.

We will look into it some more however.

**Comment by Redsandro** [comment]
I'm not sure that's right. I think Shapeshift integration is optional, and only required if you want to be able to pay bitcoins directly from altcoin wallets. Shapeshift API was integrated into Coinomi well after it already supported a bunch of altcoins.

So while Shapeshift adds another benefit to supported altcoins, shapeshiftless Coinomi-integration itself would already be quite sweet.

Thank you for looking into it.

**Comment by Redsandro** [comment]
@mjmacleod I think all we need is

* Statum server support
* [BIP44 listing](https://github.com/satoshilabs/docs/blob/master/slips/slip-0044.rst) (Hey, E-gulden is on there)

And the value of:

* `pubkey_address`
* `script_address`
* `genesis_hash`

Guldencoin needs this. As do users who don't want to have a separate wallet for every coin on their phone. :smile:

**Comment by rijkplasman** [comment]
Hi Sander,

I will make sure somebody of the team looks into this. Thanks for the info.

Kind regards,

Rijk

**Comment by Redsandro** [comment]
Thank you Rijk. This will be beneficial in multiple ways.

**Comment by Redsandro** [comment]
Updated previous post with `genesis_hash`

**Comment by Redsandro** [comment]
Coinomi is supporting some more coins now but no guldencoin.
Any progress?

**Comment by geopayme** [comment]
try https://www.cryptocompare.com/wallets/uberpay/ they have 45+ coins

**Comment by mjmacleod** [comment]
See https://github.com/Gulden/coinomi-android - I am just waiting on someone to host two electrum server instances, and for acceptance into bip44 standard, and then I will submit a pull request to coinomi.

**Comment by Redsandro** [comment]
Great! Thanks for the update. :+1:

**Comment by geopayme** [comment]
Here is the electrum server efl.el.0.uberpay.info:5037

**Comment by mjmacleod** [comment]
Sorry, I think you are confused.
This is Gulden (NLG) - efl/egulden is a different coin.

**Comment by geopayme** [comment]
good to know. :)

On Tue, Mar 8, 2016 at 10:01 AM, mjmacleod <notifications@github.com> wrote:

> Sorry, I think you are confused.
> This is Gulden (NLG) - efl/egulden is a different coin.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Gulden/gulden-official/issues/3#issuecomment-193654230>
> .
>



-- 
Geopay.me Inc.

*"don't need to know everything, I just need to know where to find it."**A.
Einstein.*
If you need to reach your people really fast, there's no better way to do
it than with "mobile".

Mobile is direct, accurate, and personal.

This email may contain confidential and/or privileged information. If you
are not the intended recipient (or have received this email by mistake),
please notify the sender immediately and destroy this email. Any
unauthorised copying, disclosure or distribution of the material in this
email is strictly prohibited.

Email transmission security and error-free status cannot be guaranteed as
information could be intercepted, corrupted, destroyed, delayed,
incomplete, or contain viruses. The sender therefore does not accept
liability for any errors or omissions in the contents of this message
which may arise as a result of email transmission.

**Comment by rijkplasman** [comment]
We are working on the two electrum servers.

**Comment by mjmacleod** [comment]
@Redsandro - Please see https://github.com/Coinomi/coinomi-android/pull/84 - it is in the Coinomi devs hands now - feel free to nag them if you want :)

**Comment by Redsandro** [comment]
This is great. I can't wait. I will urge them a bit. But as with everything open source, they too live on a planet where time passes slower than I observe the world around me.

I hope to see this implemented in the play release soon. :+1:

**Comment by mjmacleod** [comment]
Note - it appears they have added us to the wallet on play store at some point - despite the fact that there is complete silence on their github.

**Comment by geopayme** [comment]
Gulden has been added to UberPay on the play store! I have not seen any late releases of Coinomi since 2015

**Comment by mjmacleod** [comment]
Latest Coinomi release on play store "16 April 2016" adds support for Gulden. It looks like maybe they don't tag releses on Github anymore.

**Comment by geopayme** [comment]
How about that! Its a touch world after all! Good to see progress is been made!

**Comment by Redsandro** [comment]
Coinomi is under active development. I've seen a couple of releases recently on the google play beta channel.

What is UberPay? Is it seriously unrelated to Uber, UberPop and UberEats, but 'coincidentally' named like that anyway?

**Comment by geopayme** [comment]
So much hostility... no wonder!  How Uber of you "Redsandro! -  Pothead to the end!

**Comment by Redsandro** [comment]
> So much hostility... no wonder! How Uber of you "Redsandro! - Pothead to the end!

Sorry @geopayme, my autism prohibits me from understanding the true intent of your response. No wonder what? Why pothead? You used one double quote. Is it a closing or opening quote? Is your quote sarcasm because I put quotes around 'coincidentally'? I do know the connotation for pothead is a negative one, so correct me if I'm wrong but I triggered you into insulting me?

I don't get it, so let me rephrase:

_UberPay_ immediately made me think of _UberPop_, and I was excited about the notion of being able to pay my fares in not only bitcoin, but any altcoin. However, I could not find a connection to _Uber Technologies Inc._ on the _UberPay_ website, so I'm thinking there is none. Naming an app in the same theme as _UberPop_, _UberPool_ and _UberEats_, a product line from a company who's core business is _also making apps_, is a lawsuit waiting to happen when the app get's any kind of traction. Not just because because it fooled me, but also because companies are required by law to enforce their trademarks.

Please use unambiguous semantics in your response.
